stock_available_immediately/models/product_product.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2014 Camptocamp, Akretion, Numérigraphe
+# Copyright 2016 Sodexis
+# Copyright 2019 Sergio Díaz <sergiodm.1989@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _compute_available_quantities_dict(self):
+        res, stock_dict = super()._compute_available_quantities_dict()
+        for product in self:
+            if not product.is_storable:
+                res[product.id]["immediately_usable_qty"] = stock_dict[product.id][
+                    "virtual_available"
+                ]
+            else:
+                res[product.id]["immediately_usable_qty"] -= stock_dict[product.id][
+                    "incoming_qty"
+                ]
+        return res, stock_dict
+
+    @api.depends("virtual_available", "incoming_qty", "type")
```

Comment:
```suggestion
    @api.depends("incoming_qty", "is_storable")
```

- `virtual_available` already added by super()
- `is_storable` is the real dependency, not `type`